packages/lit-html/src/lit-html.ts

```diff
@@ -1373,11 +1373,11 @@ type EventListenerWithOptions = EventListenerOrEventListenerObject &
  */
 export class EventPart extends AttributePart {
   readonly type = EVENT_PART;
-  private _eventContext?: unknown;
+  private _host?: unknown;
 
   constructor(...args: ConstructorParameters<typeof AttributePart>) {
     super(...args);
-    this._eventContext = args[4]?.eventContext;
+    this._host = args[4]?.host;
```

Comment:
Since options are stored on the part now, can we eliminate the constructor? We can use `this.options.host` in `handleEvent`

Reply:
Done.